backend/server.js

```diff
@@ -11,6 +11,8 @@ const app = express();
 const FIREBASE_CONFIGS = require("./firebase.config");
 const MONGO_CONFIGS = require("./mongo.config");
 
+console.log(MONGO_CONFIGS);
```

Comment:
If we're just verifying the environment, let's only log the env name and not password or username.

Reply:
whoops forgot to remove this from debugging 👍